example/lib/month_view_page.dart

```diff
@@ -33,18 +35,18 @@ class _MonthViewPageDemoState extends State<MonthViewPageDemo> {
         controller: _controller,
         borderSize: 0.5,
         showBorder: true,
-        borderColor: Colors.blueGrey,
         cellAspectRatio: 0.55,
         pageTransitionDuration: Duration(milliseconds: 300),
         pageTransitionCurve: Curves.ease,
         cellBuilder:
             (date, List<CalendarEventData<Event>> events, isToday, isInMonth) {
           return FilledCell(
             date: date,
+            isInMonth: isInMonth,
             shouldHighlight: isToday,
             onTileTap: (event, _) =>
                 context.pushRoute(DetailsPage(event: event)),
-            backgroundColor: isInMonth ? Color(0xffffffff) : Color(0xffdedede),
+            backgroundColor: isInMonth ? Color(0xffffffff) : Color(0xffeeeeee),
```

Comment:
Can we use constants for these colors?

Reply:
These are just temporary data to test plugin. I have already updated it in new branch for plugin demo.